panel/lxqtpanel.cpp

```diff
@@ -144,14 +148,23 @@ LXQtPanel::LXQtPanel(const QString &configGroup, LXQt::Settings *settings, QWidg
     flags |= Qt::WindowDoesNotAcceptFocus;
 
     setWindowFlags(flags);
+    //Adds _NET_WM_WINDOW_TYPE_DOCK to the window's _NET_WM_WINDOW_TYPE X11 window property. See http://standards.freedesktop.org/wm-spec/ for more details.
     setAttribute(Qt::WA_X11NetWmWindowTypeDock);
+    //Enables tooltips for inactive windows.
     setAttribute(Qt::WA_AlwaysShowToolTips);
+    //Indicates that the widget should have a translucent background, i.e., any non-opaque regions of the widgets will be translucent because the widget will have an alpha channel. Setting this ...
     setAttribute(Qt::WA_TranslucentBackground);
+    //Allows data from drag and drop operations to be dropped onto the widget (see QWidget::setAcceptDrops()).
     setAttribute(Qt::WA_AcceptDrops);
 
     setWindowTitle("LXQt Panel");
     setObjectName(QString("LXQtPanel %1").arg(configGroup));
 
+    //Create the basic layout for the panel. This is only the layout that will
+    // hold the Plugins later. It is not yet the taskbar or whatever.
+    //Currently:
+    // LXQtPanel (inherits QFrame) -> QGridLayout -> LXQtPanelWidget (QFrame) -> LXQtPanelLayout
+    //Maybe, this could be shortened to LXQtPanel (inherits QFrame) -> LXQtPanelLayout some day ...
```

Comment:
I have tried it once but I forgot the reason, too, why it did not work. But then, this section needs clear documentation because these nested widgets/layouts were quite confusing when I saw them the first time.